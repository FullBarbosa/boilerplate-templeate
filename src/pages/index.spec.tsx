import { render } from '@testing-library/react';
import Home from '.';

describe('<index/>', () => {
  it('testing render componente', () => {
    const { container } = render(<Home />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
