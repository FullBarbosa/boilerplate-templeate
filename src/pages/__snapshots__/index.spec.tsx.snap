// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<index/> testing render componente 1`] = `
<div>
  index1
</div>
`;
